src/main/java/com/petfinder/domain/User.java

```diff
@@ -0,0 +1,105 @@
+package com.petfinder.domain;
+
+import java.sql.Timestamp;
+import java.util.Date;
+
+import javax.persistence.*;
+
+import org.springframework.data.jpa.domain.AbstractPersistable;
+
+@Entity
+@Table(name="users")
+public class User extends AbstractPersistable<Long>{
+   
+	@Column(name = "login", unique = true, nullable = false)
+    private String login;
+	
+	@Column(name = "email", unique = true, nullable = false)
+    private String email;
+	
+	@Column(name = "password", nullable = false)
+    private String password;
+	
+	@Column(name = "isActivated", nullable = false)
+    private boolean isActivated;
+	
+	@Column(name = "isBanned", nullable = false)
+    private boolean isBanned;
+	
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column(name = "registration_date", nullable = false)
+    private Date registration_date;
+    
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column(name = "last_login_date", nullable = true)
+    private Date last_login_date;
+	public User() {
+		super();
+	}
+    public User(String login, String email, String password){
+    	Date date= new Date();	
+
+    	this.login = login;
+    	this.email=email;
+    	this.password=password;
+    	this.isBanned = false;
+    	this.isActivated = false;
+    	this.registration_date = new Timestamp(date.getTime());
+    }
+
+	public String getLogin() {
+		return login;
+	}
+
+	public void setLogin(String login) {
+		this.login = login;
+	}
+
+	public String getEmail() {
+		return email;
+	}
+
+	public void setEmail(String email) {
+		this.email = email;
+	}
+
+	public String getPassword() {
+		return password;
+	}
+
+	public void setPassword(String password) {
+		this.password = password;
+	}
+
+	public boolean isActivated() {
+		return isActivated;
+	}
+
+	public void setActivated(boolean isActivated) {
+		this.isActivated = isActivated;
+	}
+
+	public boolean isBanned() {
+		return isBanned;
+	}
+
+	public void setBanned(boolean isBanned) {
+		this.isBanned = isBanned;
+	}
+
+	public Date getRegistration_date() {
+		return registration_date;
+	}
+
+	public void setRegistration_date(Date registration_date) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/matikz1110/PetFinder/file/2508689003/issues/source?bid=3239464&fileBranchId=3276039#l94)